doc/source/cluster/kubernetes/k8s-ecosystem/kai-scheduler.md

```diff
@@ -0,0 +1,300 @@
+
+
+## KAI Scheduler
+
+[KAI Scheduler](https://github.com/NVIDIA/KAI-Scheduler) is a high-performance, scalable Kubernetes scheduler built for AI/ML workloads. Designed to orchestrate GPU clusters at massive scale, KAI optimizes GPU allocation and supports the full AI lifecycle - from interactive development to large distributed training and inference. Some of the key features are:
+- **Bin-packing & Spread Scheduling**: Optimize node usage either by minimizing fragmentation (bin-packing) or increasing resiliency and load balancing (spread scheduling)
+- **GPU Sharing**: Allow multiple Ray workloads from across teams to be packed on the same GPU, letting your organization fit more work onto your existing hardware and reducing idle GPU time.
+- **Workload Autoscaling**: Scale Ray replicas/workers within min/max while respecting gang constraints
+- **Cluster Autoscaling**: Compatible with dynamic cloud infrastructures (including auto-scalers like Karpenter)
+- **Workload Priorities**: Prioritize Ray workloads effectively within queues
+- **Hierarchical Queues & Fairness**: Two-level queues with quotas, over-quota weights, limits and equitable resource distribution between queues using DRF
+and many more.
+For more details and key features, please refer to [the documentation](https://github.com/NVIDIA/KAI-Scheduler?tab=readme-ov-file#key-features).
+
+### Core Components
+
+1. **PodGroups**: PodGroups are atomic units for scheduling and represent one or more interdependent pods that must be executed as a single unit, also known as gang scheduling. They are vital for distributed workloads. KAI Scheduler includes a **PodGrouper** that handles gang scheduling automatically.
+
+**How PodGrouper works:**
+```
+RayCluster "distributed-training":
+├── Head Pod: 1 GPU
+└── Worker Group: 4 × 0.5 GPU = 2 GPUs
+Total Group Requirement: 3 GPUs
+
+PodGrouper ensures all 5 pods (1 head + 4 workers) are scheduled together or none at all.
+```
+
+2. **Queues**: Queues enforce fairness in resource distribution using:
+
+- Quota: The baseline amount of resources guaranteed to the queue. Quotas are allocated first to ensure fairness.
+- Queue Priority: Determines the order in which queues receive resources beyond their quota. Higher-priority queues are served first.
+- Over-Quota Weight: Controls how surplus resources are shared among queues within the same priority level. Queues with higher weights receive a larger share of the extra resources.
+- Limit: Defines the maximum resources that the queue can consume.
+
+Queues can be arranged hierarchically for organizations with multiple teams (e.g. departments with multiple teams).
+
+## Prerequisites
+
+* Kubernetes cluster with GPU nodes
+* NVIDIA GPU Operator 
+* kubectl configured to access your cluster
+
+## Step 1: Install KAI Scheduler
+
+Install KAI Scheduler with gpu-sharing enabled:
```

Comment:
Would you mind to put KAI-scheduler release page link in this section(Install KAI Scheduler) to help the user find the version easily ?